Rakefile.rb

```diff
@@ -85,20 +103,33 @@ def prepare_and_execute_build(integration, dont_error_on_build: false)
     'name' => "#{integration}",
     'version' => "#{integration_version}",
     'build_iteration' => "#{ENV['BUILD_ITERATION']}",
-    'integrations_repo' => "#{ENV['INTEGRATIONS_REPO']}"
+    'integrations_repo' => "#{ENV['INTEGRATIONS_REPO']}",
+    'guid' => "#{manifest['guid']}",
+    'description' => "#{manifest['description']}"
   })
 
-  sh "(echo '#{header}' && cat #{PROJECT_DIR}/resources/datadog-integrations/project.rb.erb) | erb > #{PROJECT_DIR}/config/projects/dd-check-#{ENV['INTEGRATION']}.rb"
+  `(echo '#{header}' && cat #{PROJECT_DIR}/resources/datadog-integrations/project.rb.erb) | erb > #{PROJECT_DIR}/config/projects/dd-check-#{integration}.rb`
```

Comment:
`sh` is a rake DSL if I remember correctly, could a`system` call work instead? (The backticks syntax is not very Ruby-beginner friendly)